packages/mobile/src/components/countdown/components/box/styles.ts

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  box: {
+    width: 67,
+    height: 67,
+    marginRight: 10,
+  },
+  number: {
+    fontFamily: 'Roboto',
```

Comment:
Why are we using different font here?

Reply:
This font taken from layout

Reply:
![Font](https://user-images.githubusercontent.com/69375560/188085251-7293f5a4-fc1e-4904-8fd1-2b7ebe2401be.jpg)